### PR TITLE
Minimal HW manifest support need for FMUv5 

### PR DESCRIPTION
**Pull request by davids5**
Platforms that provide HW versioning tend to have more FLASH and can have a PX4IO as an option.
   (E.G.) FMUv5. This change provides run time  decision to control the safety switch on
   platforms that have BOARD_HAS_VERSIONING and support a manifest. (FMUv5)

   On platforms with no versioning and have BOARD_HAS_STATIC_MANIFEST defined as a 1
   The decision is compile time only. When a PX4IO is present the fmu control of the safety is
   not built.

### CODE REVIEW

File: src/drivers/boards/px4fmu-v2/manifest.c [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+static const px4_hw_mft_item_t hw_mft_list_v9000e[] = {
+	{
+		.present     = 1,
+		.mandatory   = 1,
+		.connection  = px4_hw_con_onboard,
+	},
+};
+
+static px4_hw_mft_list_entry_t mft_lists[] = {
+	{0x90008, hw_mft_list_v90008, arraySize(hw_mft_list_v90008)},
```
Comment (dagar) [inline src/drivers/boards/px4fmu-v2/manifest.c]: What are these numbers?

Comment (davids5) [reply src/drivers/boards/px4fmu-v2/manifest.c]: @dagar I will push a clean up.

**Comment by Avysuarez** [comment]
tested with pixhawk 4 (V5).
safety switch working well.
https://review.px4.io/plot_app?log=533ab0d1-2a87-48e9-bb79-d09d5b33cdd7

**Comment by dannyfpv** [comment]
test on pixhawk mini v3 work well.
https://review.px4.io/plot_app?log=af81d7ab-4917-4168-8981-f2064890d43d

pixhawk pro v4 work well
https://review.px4.io/plot_app?log=eb3abd98-b9e6-4320-b9f7-ef44932f406a

**Pull request merged**